### ISSUE TITLE
Allow user to upload their own profile image

### ISSUE THREAD

**Issue by lazlo28** [snoyberg/haskellers#32]
As now the only choice you have is to use gravatar, which for a number of reasons (not wanting to use a single profile pic for x sites, etc.) may not be the best choice.

So an idea would be to allow users to uplod their own profile pic if they want so.

**Comment by snoyberg** [comment]
I'd accept a PR that lets users point to externally hosted images, but I
don't want to add image hosting overhead to Haskellers.

**Comment by lazlo28** [comment]
Correct, images hosted elsewhere would do, too. I updated the title of the issue appropriately.
